images/macos/tests/BasicTools.Tests.ps1

```diff
@@ -144,7 +144,7 @@ Describe "vagrant" -Skip:($os.IsBigSur) {
     }
 }
 
-Describe "virtualbox" -Skip:($os.IsBigSur) {
+Describe "virtualbox" {
```

Comment:
nitpick: virtualbox -> VirtualBox

Reply:
Let's add test to validate kextstat at the end of the build

Reply:
Done